packages/cli/src/cmds/index/index.ts

```diff
@@ -141,13 +141,13 @@ export const handler = async (argv) => {
 
       const rpcMethods: RpcHandler<any, any>[] = [
```

Comment:
@dustinbyrne I think what we need to do here is set the Configuration from the CLI parameters. That way, older versions of the extension that are still launching `index -p -w` (an indexer + RPC per-process) will continue to work. We can do that until everyone in the wild has upgraded beyond extension versions that still do that.

Reply:
This is for testing and backwards compatibility. 

The IDE extensions should launch without the directory argument and set configuration as soon as they have the port number.